### PR TITLE
Add rich property editor support for .NET Core projects

### PR DESCRIPTION
**Pull request by DVaughan**
### Description of Change ###

Added support for rich property editors in the Properties view.

### API Changes ###

None

### Behavioral Changes ###

Specialized property editors are now displayed in the Properties view in Blend.

### CODE REVIEW

File: src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs [hunk 1]
```diff
@@ -12,16 +13,16 @@ namespace Microsoft.Xaml.Behaviors.DesignTools
 {
     internal class MetadataTableProvider : IProvideAttributeTable
     {
-        private AttributeTableBuilder _attributeTableBuilder;
+        private AttributeTableBuilder attributeTableBuilder;
```
Comment (brianlagunas) [inline src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs]: why remove the underline from the private variable?

File: src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs [hunk 2]
```diff
@@ -12,16 +13,16 @@ namespace Microsoft.Xaml.Behaviors.DesignTools
...
 
         // Accessed by the designer to register any design-time metadata.
         public AttributeTable AttributeTable
         {
             get
             {
-                if (_attributeTableBuilder == null)
+                if (this.attributeTableBuilder == null)
```
Comment (mgoertz-msft) [inline src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs]: In this repo we have different conventions: no this.

File: src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs (hunk 1 above)
Comment (DVaughan) [reply src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs]: Restored the underscore.

File: src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs (hunk 2 above)
Comment (DVaughan) [reply src/Microsoft.Xaml.Behaviors.DesignTools/MetadataTableProvider.cs]: Removed.

**Pull request merged**